CMakeLists.txt

```diff
@@ -403,9 +388,11 @@ IF(NOT WIN32)
 ENDIF()
 target_link_libraries( fc PUBLIC ${LINK_USR_LOCAL_LIB} ${Boost_LIBRARIES} ${OPENSSL_LIBRARIES} ${ZLIB_LIBRARIES} ${PLATFORM_SPECIFIC_LIBS} ${RPCRT4} ${CMAKE_DL_LIBS} ${rt_library} ${editline_libraries} ${ECC_LIB} )
 
-if(MSVC)
-  set_source_files_properties( src/network/http/websocket.cpp PROPERTIES COMPILE_FLAGS "/bigobj" )
-endif(MSVC)
+if(WIN32 AND MSVC)
+    set_source_files_properties( src/network/http/websocket.cpp PROPERTIES COMPILE_FLAGS "/bigobj" )
```

Comment:
I do not believe /bigobj is necessary here unless you are building in debug.

Reply:
This extends the behaviour that was already there to support the same for MinGW compilation.